### PR TITLE
Add credentials to pipelines' roadmap

### PR DESCRIPTION
**Pull request by sbwsg**
# Changes

Getting credentials into Tasks is currently supported via multiple
mechanisms, each with its own set of tradeoffs and level of support
within Tekton Pipelines.

This PR adds credentials to our roadmap with intent to improve both upon
the implemented feature set and the documented guidance to credential handling
that Pipelines already offers.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

### CODE REVIEW

**Comment by bobcatfish** [comment]
Thanks @sbwsg ! +1 from me - would be great to get a couple more OWNERS to lgtm before we merge since it's a roadmap change

@ImJasonH @afrittoli @dlorenc @dibyom 

im also not sure if the meow means you agree or not @vdemeester but im guessing it does XD

File: roadmap.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Tekton:
 - [Adding support for other architectures](https://github.com/tektoncd/pipeline/issues/856)
 - [Rich type support for params](https://github.com/tektoncd/pipeline/issues/1393)
 - Looping syntax (in [Tasks](https://github.com/tektoncd/pipeline/issues/2112) and/or [Pipelines](https://github.com/tektoncd/pipeline/issues/2050))- either implement or decide it is outside of scope (i.e. better suited for a DSL)
+- [Supplying credentials for authenticated actions in Tasks](https://github.com/tetkoncd/pipeline/issues/2343) - documented guidelines
```
Comment (dibyom) [inline roadmap.md:46]: typo: tekton :D

**Comment by dibyom** [comment]
Strong +1 to adding this from me!
/lgtm

**Comment by vdemeester** [comment]
ahah yes I do agree :wink:

File: roadmap.md (hunk 1 above)
Comment (sbwsg) [reply roadmap.md]: hand wrote the url 😱 fixed.  thanks!

**Comment by bobcatfish** [comment]
I'm gonna remove the hold - if has strong feelings that they haven't expressed yet we can always modify :D

/hold cancel

**Pull request merged**